dev/com.ibm.websphere.appserver.features/visibility/public/data-1.0/resources/l10n/io.openliberty.data-1.0.properties

```diff
@@ -16,4 +16,8 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=Jakarta Data provides a repository-based data access pattern where the application defines custom data repository interfaces and the container/runtime provides the implementation.
\ No newline at end of file
+description=Jakarta Data is a programming model for data access that\
+ allows you to represent data with simple Java objects called entities\
+ and define operations on data as methods of a repository interface.\
+ A Jakarta Data provider supplies the implementation. This feature\
+ includes a built-in Jakarta Data provider for relational database access.
```

Comment:
Jakarta Data is a programming model for data access.
You can use Jakarta Data to represent data with Java objects called entities and to define operations in data as methods of a repository interface.
A Jakarta Data provider supplies the repository implementation to an application. 
The data-1.0. feature includes a built-in Jakarta Data provider for relational database access.